content/iOS/RSFormView/RSFormViewExample.md

```diff
@@ -289,10 +299,10 @@ func submitButtonTapped() {
   present(alert, animated: true)
 }
 ```
-`switchValueChanged` changes our views aspect according to the selected value, if set to on we go for a "dark mode" otherwise we go for a clear mode. 
+`switchValueChanged` changes our views aspect based on the selected value. If set to on set the "dark mode" otherwise we go set the default mode. 
```

Comment:
I don't understand the corrected sentence, I think the original was clearer. It could also be: `If set to on, they turn to a "dark mode", otherwise they use a "clear mode".` Or something like that.

Reply:
agreed, there's a missing comma there. Good catch! :)